### ISSUE TITLE
[LOW] Add Cross-Validation Scores to Individual Notebooks

### ISSUE THREAD

**Issue by lucidopus** [PradHolla/CS559-Final-Project#9]
## Enhancement: Display Cross-Validation Performance

While not explicitly required, showing cross-validation performance provides valuable insights and demonstrates model robustness.

## Suggested Additions

For each subgroup model, add section displaying:
- [ ] Cross-validation scores for Eq(1) accuracy (recall on y=1)
- [ ] Show mean ± std across folds
- [ ] Helps justify model selection and demonstrates generalization capability

## Example Output
```
Cross-Validation Results (5-fold StratifiedKFold, Recall on Bankrupt):
  Mean: 0.766 ± 0.089
  Scores: [0.769, 0.778, 0.630, 0.769, 0.885]
  Min: 0.630, Max: 0.885
```

## Implementation Hint
```python
from sklearn.model_selection import cross_val_score
from sklearn.metrics import recall_score, make_scorer

recall_scorer = make_scorer(recall_score, pos_label=1, zero_division=0)

cv_scores = cross_val_score(
    model_pipeline,
    X_sub,
    y_sub,
    cv=cv_strategy,
    scoring=recall_scorer,
    n_jobs=-1
)

print(f'Mean: {cv_scores.mean():.4f} ± {cv_scores.std():.4f}')
print(f'Scores: {cv_scores}')
```

## Files to Update
- [ ] `[PersonA]_0.ipynb` - Add CV section
- [ ] `[PersonB]_4.ipynb` - Add CV section
- [ ] `[PersonC]_2.ipynb` - Add CV section
- [ ] `[PersonD]_2.ipynb` - Add CV section

## Acceptance Criteria
- [ ] CV scores shown for each individual model
- [ ] Metric used: Recall (Eq(1) accuracy)
- [ ] Mean and std reported
- [ ] Shows robustness across folds
